doc/cephfs/lazyio.rst

```diff
@@ -21,3 +21,57 @@ LazyIO can be enabled by following ways.
 
 - ``ioctl(fd, CEPH_IOC_LAZYIO, 1UL)`` enables LAZY_IO for file handle in
    ceph-fuse mount.
+
+Using LazyIO
+============
+
+ LazyIO includes two methods ``lazyio_propagate()`` and ``lazyio_synchronize()``.
+ With LazyIO enabled, writes may not be visble to other clients until
+ ``lazyio_propagate()`` is called. Reads may come from local cache (irrespective of
+ changes to the file by other clients) until ``lazyio_synchronize()`` is called.
+
+- ``lazyio_propagate(...)`` - Ensures that any buffered writes of the calling client,
+  in the specific region, has been propagated to the shared file.
+
+- ``lazyio_synchronize(...)`` - Ensures that the calling client is, in a subsequent
+  read call, able to read the updated file with all the propagated writes of the 
+  other clients. In CephFS this is facilitated by invalidating the inode of the 
```

Comment:
invalidating the file page cache**

Reply:
But page cache is for the kernel client right? Should I say "cached inode" ?

Reply:
The fuse driver permits the kernel to cache file pages too (this can be turned off). The "file page cache" is not specific to the kernel though and is fairly common term for the file data blocks in memory.